methods/add_student.rb

```diff
@@ -0,0 +1,12 @@
+def add_student
+  print 'Enter the name of the student: '
+  name = gets.chomp.to_s
+  print 'Enter the age of the student: '
+  age = gets.chomp.to_i
+  print 'Has parent permission? [Y/N]: '
```

Comment:
- [ ] Following the principle of single responsibility, please create a method whose sole responsibility is to request and validate the student input